src/OrchardCore.Modules/OrchardCore.Workflows/WorkflowPruning/AdminMenu.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Navigation;
+using OrchardCore.Workflows.WorkflowPruning.Drivers;
+
+namespace OrchardCore.Workflows.WorkflowPruning;
+
+public sealed class AdminMenu : INavigationProvider
+{
+    private static readonly RouteValueDictionary _routeValues = new()
+    {
+        { "area", "OrchardCore.Settings" },
+        { "groupId", WorkflowPruningDisplayDriver.GroupId },
+    };
+
+    internal readonly IStringLocalizer S;
+
+    public AdminMenu(IStringLocalizer<AdminMenu> stringLocalizer)
+    {
+        S = stringLocalizer;
+    }
+
+    public Task BuildNavigationAsync(string name, NavigationBuilder builder)
+    {
+        if (!NavigationHelper.IsAdminMenu(name))
+        {
+            return Task.CompletedTask;
+        }
+
+        builder.Add(S["Configuration"], configuration => configuration
+            .Add(S["Settings"], settings => settings
+                .Add(S["Workflow Pruning"], S["Workflow Pruning"], pruning => pruning
```

Comment:
Any particular reason to call this "pruning" vs "trimming" what we have for Audit Trail?

Reply:
Likely just a comment from Seb on the original issue

Reply:
I think trimming would be better, not to have a new term for the same concept.